pom.xml

```diff
@@ -251,6 +251,11 @@
 			<artifactId>commons-collections4</artifactId>
 			<version>4.3</version>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-text</artifactId>

```

Comment:
This is fine but you may consider trying out https://jsoup.org/cookbook/cleaning-html/whitelist-sanitizer 

You can set it to reject all html. Not sure if this will also cover javascript escaping?